packages/amplify-cli/src/packageManagerHelpers.ts

```diff
@@ -24,6 +25,28 @@ export async function getPackageManager() {
   return undefined;
 }
 
+/**
+ * Determine the starting command of the current project
+ *
+ * @return {string} 'serve' or 'start'
+ */
+export async function getPackageManagerCommand() {
+  let scripts;
+  if (fs.existsSync(packageJsonDir)) {
+    const pjson = fs.readFileSync(packageJsonDir).toString();
+    const parsedPjson = JSON.parse(pjson);
+    scripts = parsedPjson['scripts'];
+  }
+
+  if (scripts['start']) {
```

Comment:
if package.json doesn't exist or doesn't contain a "scripts" field this will error

Reply:
Changed to check if scripts exists and moved into package.json check.